traffic_stats/traffic_stats.go

```diff
@@ -555,7 +555,9 @@ func loadStartupConfig(configFile string, oldConfig StartupConfig) (StartupConfi
 	//Close old connections explicitly
 	for _, host := range oldConfig.InfluxDBs {
 		if host.InfluxClient != nil {
-			host.InfluxClient.Close()
+			if err := host.InfluxClient.Close(); err != nil {
+				errorf("closing influx client: %s", err)
```

Comment:
Unfortunately, no, because Traffic Stats is currently straddling two different logging libraries (lib/go-log and seelog), hence why we have to use an `errorf` helper func here. Once we get rid of seelog support, we can change these to `log.Close(...)`.